app/assets/javascripts/models/data/service_options.js.coffee

```diff
@@ -119,7 +118,12 @@ ns.ServiceOptions = do (ko, edsc = @edsc, KnockoutModel = @edsc.models.KnockoutM
                 if !accessMethodMatched && matches[1] == jsonObj.collection_id && parseInt(matches[2], 10) == index
                   accessMethodMatched = true
                   @loadForm(true) if jsonObj.type == 'service'
-                  ko.applyBindingsToNode(echoformContainer, {echoform: this})
+                  # After the binding is complete, toggle select elements to trigger validation lest they be pruned
+                  $.when(ko.applyBindingsToNode(echoformContainer, echoform: this)).done ->
```

Comment:
@mightynimble @mreese84  - I think this'll do it.  I realized that projection select dropdowns are always going to have 'NORTH POLAR STEREOGRAPHIC' as an option - so why not just leverage that?  One line fix.  This new version only toggles selects with that option available.